proxy/src/proxy/wake_compute.rs

```diff
@@ -64,55 +62,7 @@ fn report_error(e: &WakeComputeError, retry: bool) {
     let kind = match e {
```

Comment:
Thinking about it, I think we should just remove this kind - or at least just use the cplane reason verbatim. This error kind pre-dated the cplane reason and is a lot less useful now

Reply:
Do you mean the whole WakeupFailureKind, right?